src/components/PeopleFilters.tsx

```diff
@@ -16,77 +43,50 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={query}
+            onChange={(event) => setCurrentQuery(event.target.value)}
           />
 
           <span className="icon is-left">
-            <i className="fas fa-search" aria-hidden="true" />
+            <i
+              className="fas fa-search"
+              aria-hidden="true"
+              onClick={deleteQuery}
```

Comment:
The 'onClick' event should be placed on a button element instead of the 'i' element for better accessibility